src/mjolnir/convert_transit.cc

```diff
@@ -264,17 +232,6 @@ ProcessStopPairs(GraphTileBuilder& transit_tilebuilder,
             dow_mask = kDOWNone;
           }
 
-          // if dep.days == 0 then feed either starts after the end_date or tile_header_date >
-          // end_date
-          if (days == 0 && !stop_pair.service_added_dates_size()) {
-            LOG_ERROR("Feed rejected: " + pbf_fp +
-                      "!  Start date: " + to_iso_extended_string(start_date) +
-                      " End date: " + to_iso_extended_string(end_date));
-            continue;
-          }
```

Comment:
this was crazy verbose! in the feed I have, this happens frequently, lots of added dates to empty schedules are in january.